src/services/user.service.ts

```diff
@@ -24,7 +24,7 @@ export class UserService {
   }
 
   // user will be null if there isn't
-  // this email in db => account can be registered
+  // that email in db => account can be registered
```

Comment:
I'd remove this comment - first I don't quite understand it 😅 , second, method is readable enough to see what it does